SQL/migrate/V046_IPC_tag_project.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Reworks how IPC tags work in PR #6973.
+-- This drops the old ss13_ipc_tracking table in favour of ss13_characters_ipc_tags with new fresh information.
+-- 
+
+DROP TABLE `ss13_ipc_tracking`;
+
+CREATE TABLE `ss13_characters_ipc_tags` (
+  `char_id` int(11) unsigned NOT NULL,
+  `tag_status` tinyint(1) NOT NULL DEFAULT '0',
+  `serial_number` varchar(12) DEFAULT NULL,
+  `ownership_status` varchar(16) DEFAULT NULL,
```

Comment:
If there are only a fixed number of options for the ownership, then the ownership should be a enum and not a varchar